src/toniebox_state.c

```diff
@@ -82,7 +97,6 @@ void tbs_playback(client_ctx_t *client_ctx, toniebox_state_playback_t playback)
     default:
         break;
     }
-    mqtt_sendBoxEvent("TagInvalid", "", client_ctx);
```

Comment:
We can't remove this, as we don't have the event for the CC3235, this would break the functionality for it.